README.md

```diff
@@ -1,3 +1,86 @@
-### module-1-hello-project
-This project is created to practice workflow in GitHub and use of Linters
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="cover-image.png" alt="logo" height="100" width="auto" />
+  <br/>
+
+  <h3><b>Module 1 Hello Microverse</b></h3>
+  <br/>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[About the project\] ](#-about-the-project-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [About the project] <a name="about-project"></a>
+
+This project is created to practice workflow in GitHub, pull Request and Linters
+
+**Hello Microverse** is a Hello Microverse web page to practice use of linters
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Use a style.css file to beautify the page.
+
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> there's no live demo yet.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> You can follow the Microverse repo's about linter to learn about this topic
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@jicamargo](https://github.com/jicamargo)
+- Twitter: [@ji_camargo](https://twitter.com/ji_camargo)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/jorgecamargog)
+
+👤 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<!-- SUPPORT -->
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project i will pray for you! 🙏
+
+<!-- ACKNOWLEDGEMENTS -->
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> I would like to thank my code partners
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
Very good job so far and well done on applying changes requested by the previous reviewer , one small but important thing is still missing is the Licence section it is important for legal reasons don't forget to add it and to put a valid link to your licence file

Reply:
thank you @souad988, I added it to readme.md file